react-effects/my-app/src/App.tsx

```diff
@@ -0,0 +1,6 @@
+import List from './List.tsx';
```

Comment:
You should delete the `.jsx` files when you bring in the `.tsx` files so that the original import (`'./List'`) will work.